yarn.lock

```diff
@@ -4198,6 +4198,14 @@ fs-extra@^5.0.0:
     jsonfile "^4.0.0"
     universalify "^0.1.0"
 
+fs-extra@^6.0.1:
```

Comment:
Why did this update fs-extra and request? I only updated `elife-theme` version in `package.json` and ran `yarn`. `elife-theme` only depends on `styled-components`.

Reply:
I think running yarn will do minor updates if the lockfile needs updating

Reply:
This answer it: 

```
$ git blame server/submission/package.json
7dfff4d1 (Mihail Dunaev 2018-06-01 14:40:34 +0100 14)     "fs-extra": "^6.0.1",
```

Remember the root package.json allows > 4.0.2